File: benchmark/2026-09-17-7677581(latest)/standard/rubrics/probes/CONC-2/go/conc2.go
// CONC-2: two concurrent workers increment a shared counter with NO synchronization.
package main

import (
	"fmt"
	"sync"
)

const ITERS = 200000

var counter int64

func main() {
	var wg sync.WaitGroup
	for k := 0; k < 2; k++ {
		wg.Add(1)
		go func() {
			defer wg.Done()
			for i := 0; i < ITERS; i++ {
				counter = counter + 1
			}
		}()
	}
	wg.Wait()
	fmt.Printf("counter=%d expected=%d\n", counter, 2*ITERS)
}

File: benchmark/2026-09-17-7677581(latest)/standard/rubrics/probes/CONC-2/swift/conc2.swift
// CONC-2: two concurrent workers increment a shared counter with NO synchronization.
import Foundation
import Dispatch

let ITERS = 200000
var counter: Int = 0

let group = DispatchGroup()
for _ in 0..<2 {
    DispatchQueue.global().async(group: group) {
        for _ in 0..<ITERS { counter = counter + 1 }
    }
}
group.wait()
print("counter=\(counter) expected=\(2 * ITERS)")
